editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/document_node_types.rs

```diff
@@ -110,6 +110,26 @@ static DOCUMENT_NODE_TYPES: &[DocumentNodeType] = &[
 		outputs: &[FrontendGraphDataType::Raster],
 		properties: node_properties::gpu_map_properties,
 	},
+	DocumentNodeType {
+		name: "Blur",
+		category: "Image Adjustments",
+		identifier: NodeIdentifier::new("graphene_core::raster::BlurNode", &[]),
+		inputs: &[
+			DocumentInputType::new("Image", TaggedValue::Image(Image::empty()), true),
+			DocumentInputType::new("Radius", TaggedValue::U32(3), false),
```

Comment:
Use `F64` (and round the value) and add a TODO to change this back to U32 once we have a working dynamic type system. But also this should probably be allowed to be a decimal value anyways, I think that's mathematically possible. (For example, in the FFT -> Blur in Frequency Domain -> IFFT approach, that should allow a decimal blur radius, and surely there's an equivalent for the spatial domain blur approach.)